src/providers/dataProvider/dataprovider-utils.ts

```diff
@@ -66,6 +66,12 @@ export const auditForUpdatedChanges = async (
   subject?: User['id']
 ): Promise<UpdateParams<RCOResource>> => {
   const difference = getDifference(record.data, record.previousData)
+  const keys = Object.keys(difference)
```

Comment:
@salony75way - I'm nervous that a subsequent change to dispatchedAt or reportPrintedAt could report that the previous values were null, when they weren't.  
The current UI prevents this happening, but we don't know what will happen with the UI in the future.  So, I'd rather we check that there was no value previously present, please.

Reply:
Done!